imports/plugins/core/files/server/fileCollections.js

```diff
@@ -10,6 +10,7 @@ import {
   RemoteUrlWorker,
   TempFileStoreWorker
 } from "@reactioncommerce/file-collections";
+import { AbsoluteUrlMixin } from "/imports/plugins/core/core/server/Reaction/absoluteUrl";
```

Comment:
importing the mixin directly to avoid a circular dependency.
I have found that circular dependencies are _eventually_ resolved once both files have been fully loaded. We rarely notice them (if they exist) because most calls are made in some initialization step once all of the code is loaded. this, on the other hand, is being called immediately, before the `Reaction` API has not yet been fully loaded.